<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridToolbarWidget</class>
 <widget class="QWidget" name="GridToolbarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Units:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="unitsBox">
     <item>
      <property name="text">
       <string>mil</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mm</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Grids: Visible</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="visGridBox">
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25 mm</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Placement</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="placeGridBox">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>125</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.1 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.2 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.25 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 mm</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Routing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="routGridBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.333</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6.666</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16.666</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.01 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.02 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.04 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.05 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.1 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.2 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.25 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.4 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 mm</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="angleBox">
     <item>
      <property name="text">
       <string>45°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
